PowerUp/app/src/main/java/powerup/systers/com/powerup/PowerUpUtils.java

```diff
@@ -307,5 +307,17 @@
             R.drawable.vocab_skinny,
             R.drawable.vocab_tampon
     };
+
+    public static boolean[] VIRUS_HIT = {
```

Comment:
The array here is used to save the virus that were killed previously and set the visibilities accordingly when the game is relaunched.